main.go

```diff
@@ -117,50 +137,85 @@ func main() {
 		editLock      = args["-k"].(bool)
 	)
 
-	initlog(args["--debug"].(bool), args["--trace"].(bool))
+	log.Init(args["--debug"].(bool), args["--trace"].(bool))
 
 	config, err := LoadConfig(filepath.Join(os.Getenv("HOME"), ".config/mark"))
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	markdownData, err := ioutil.ReadFile(targetFile)
+	creds, err := GetCredentials(args, config)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	meta, err := mark.ExtractMeta(markdownData)
+	api := confluence.NewAPI(creds.BaseURL, creds.Username, creds.Password)
+
+	markdown, err := ioutil.ReadFile(targetFile)
 	if err != nil {
 		log.Fatal(err)
 	}
 
-	if dryRun {
-		fmt.Println(string(mark.CompileMarkdown(markdownData)))
-		os.Exit(0)
+	meta, err := mark.ExtractMeta(markdown)
+	if err != nil {
+		log.Fatal(err)
 	}
 
-	creds, err := GetCredentials(args, config)
+	stdlib, err := stdlib.New(api)
```

Comment:
can we do something less abstract? maybe longer name but more obvious about purposes of the package

Reply:
I can't come up with a better name. Do you have some variants in mind?